packages/frontend/src/components/Info/Accordion.tsx

```diff
@@ -61,7 +76,7 @@ export const InfoAccordion = () => {
           </StyledHeader>
           <StyledContent>
             The ticket is an NFT token held in your wallet. A unique token will be minted in the same moment you make a
```

Comment:
```suggestion
            The ticket is an NFT held in your wallet. A unique token will be minted in the same moment you make a
```
Since otherwise it says "non-fungible token token"
Also is this accurate/up-to-date info?

Reply:
I would suggest to change it to "If your bet wins a ticket, a unique voucher code will be generated for it. Enter this code in the sales system and follow the instructions to receive your ticket." 
But I am not sure about the exact form of a ticket.